packages/vue-vuetify/src/complex/AllOfRenderer.vue

```diff
@@ -20,7 +20,7 @@
         />
         <dispatch-renderer
           v-for="(allOfRenderInfo, allOfIndex) in allOfRenderInfos"
-          :key="`${control.path}-${allOfIndex}`"
+          :key="`${control.path}-${allOfRenderInfos.length}-${allOfIndex}`"
```

Comment:
Why adapt this key?

Reply:
because if the size of the allOfRendererInfo changes then the key is not stable enough to guarantee that previous keys will point the exact same elements - e.g. we do not have a primary key for the info that we can use - usually this happens when we change the UI schema and the schema have to be applied again.

Reply:
I agree as well but then we do need that on the core level - for now at least that is fixing the issue that I was having when I change the uischema from the monaco editor in our vue examples. So once this is available at the core - maybe some id for the uischema changes then we can utilize that.